plutus-conformance/test-cases/uplc/evaluation/failure/cons-bs-arg-overflow/cons-bs-arg-overflow.uplc

```diff
@@ -0,0 +1,2 @@
+-- the arg overflow'ed over the maxBound :: Word8
+(program 1.0.0 [(builtin consByteString) (con integer 256) (con bytestring #)])
```

Comment:
I think we have a specific builtins folder for behaviour of specific builtins, which I think this is?

Reply:
and some more below

Reply:
so we have a `builtins` folder and a `failure` folder. Not sure if it's a sensible organization? But if it is, then failures should all go in the `failure` folders.

Reply:
I'm not sure that "failure" is such an interesting categorization. I think people are more likely to be interested in a class of behaviour, and then they're going to be interested in both successful and failing behaviour. so my money goes for "just put them all in the builtins folder"

Reply:
Sounds good to me.